src/components/card/event-card/index.style.js

```diff
@@ -0,0 +1,107 @@
+import styled from 'styled-components';
+import { Link as ReactRouterLink } from 'react-router-dom';
+
+import CardTemplate from '../card-template';
+import { Text, Heading } from '../../typography';
+import Icon from '../../icons';
+
+
+/* Common event card styles */
+
+export const Link = styled(ReactRouterLink)`
+  text-decoration: none;
+`;
+
+export const DateBookmarkContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+`;
+
+export const BookmarkIcon = styled(Icon)`
+  margin-left: 0;
+  &:hover {
+    cursor: pointer;
+  }
+`;
+
+export const Title = styled(Heading)`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+`;
+
+export const PlacePriceIcon = styled(Icon)`
+  margin-right: 0;
+`;
+
+
+/* Vertical event card styles */
+
+
+export const VerticalCardContainer = styled(CardTemplate)`
+  margin: 8px;
+`;
+
+export const VerticalCover = styled.img`
+  width: 100%;
+  height: auto;
+`;
+
+export const VerticalContentContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 16px;
+  height: 200px;
```

Comment:
this file has so many hard-coded numbers that can come from a config file like a theme file, so when we want to develop an element in react-jupiter later, just need to change them and plus the numbers cant be so special (like 200px or 206px) and manage them to be standard.